### PR TITLE
docs: Tidy the navbar

### PR DESCRIPTION
**Pull request by Callisto13**
And break up 'Getting Started'. This layout is inspired by Tilt's docs, which are often mentioned as Good. The sidebar here is set to always expanded so they are always visible by default.

image (27)

This follows changes in https://github.com/weaveworks/weave-gitops/pull/3332 which crowded the navbar and can go in after https://github.com/weaveworks/weave-gitops/pull/3328

Part of https://github.com/weaveworks/weave-gitops-enterprise/issues/2274

The navbar is not versionable which is a right pain, so either we never touch the navbar, or we live with it 🤷‍♀️ .

### CODE REVIEW

File: website/sidebars.js [hunk 1]
```diff
@@ -11,31 +11,58 @@
...
+          collapsed: false,
+          link: {
+            type: 'doc',
+            id: 'getting-started/intro',
+          },
+          items: [
+            {
+              type: 'doc',
+              id: 'installation/index',
+              label: '0. Install Weave GitOps',
```
Comment (makkes) [inline website/sidebars.js:58]: I know we're all nerds but are our customers fine with starting to count from 0?

File: website/sidebars.js [hunk 2]
```diff
@@ -11,31 +11,58 @@
...
       items: [
-        'installation/weave-gitops',
         {
           type: 'category',
-          label: 'Weave GitOps Enterprise',
+          label: 'Installation',
           link: {
             type: 'doc',
-            id: 'installation/weave-gitops-enterprise/index',
+            id: 'installation/index',
```
Comment (makkes) [inline website/sidebars.js:28]: Putting my comment here as the page in question isn't changed as part of this PR: The page's title should follow a similar pattern as the "part 1" and "part 2" pages and have a title preceded by "Part 0: ".

Comment (Callisto13) [reply website/sidebars.js:28]: it does and doesn't make sense to do that.

the installation section is still separate from the getting started section, the getting started just links back to installation in its first.

if i were to pull installation under getting started, then yes having it be part 0 makes sense (i can even 1-index more naturally there). but if we want installation to remain separate (and there is a good case for doing that), then it makes less sense to have titled with a number for people who are coming just to the installation bit

Comment (Callisto13) [reply website/sidebars.js:28]: image (5)

Looks like this now

**Review by makkes — APPROVED** [review]
Love it! ✨

**Pull request merged**